examples/kitchen-sink/__test__/build.test.js

```diff
@@ -1,48 +1,40 @@
 const fs = require('fs')
 
+const expected = [
+  'async.css',
+  'financial-times-o-ads.bundle.js',
+  'financial-times-o-grid.bundle.js',
+  'financial-times-o-tracking.bundle.js',
+  'financial-times-o-utils.bundle.js',
+  'financial-times-o-viewport.bundle.js',
+  'manifest.json',
+  'o-footer.bundle.js',
+  'o-grid.bundle.js',
+  'o-header.bundle.js',
+  'o-toggle.bundle.js',
+  'o-typography.bundle.js',
+  'o-utils.bundle.js',
+  'o-viewport.bundle.js',
+  'page-kit-components.bundle.js',
+  'page-kit-layout-styles.css',
+  'regenerator-runtime.bundle.js',
+  'scripts.bundle.js',
+  'shared.stable.bundle.js',
+  'shared.volatile.bundle.js',
+  'styles.css',
+  'webpack-runtime.bundle.js'
+]
+
 describe('examples/kitchen-sink/build', () => {
   let output
 
   beforeAll(() => {
     output = fs.readdirSync('./public')
   })
 
-  it('creates a JS bundle for each Origami component', () => {
-    expect(output).toContain('o-header.bundle.js')
-    expect(output).toContain('o-footer.bundle.js')
-    expect(output).toContain('o-utils.bundle.js')
-    expect(output).toContain('o-viewport.bundle.js')
-  })
-
-  it('creates a JS bundle for Page Kit UI components', () => {
-    expect(output).toContain('page-kit-components.bundle.js')
-  })
-
-  it('creates a shared JS bundle', () => {
-    expect(output).toContain('shared.stable.bundle.js')
-  })
-
-  it('creates bundles for Webpack runtime', () => {
-    expect(output).toContain('webpack-runtime.bundle.js')
-  })
-
-  it('creates an app JS bundle', () => {
-    expect(output).toContain('scripts.bundle.js')
-  })
-
-  it('creates an app CSS bundle', () => {
-    expect(output).toContain('styles.css')
-  })
-
-  it('creates an app async CSS bundle', () => {
-    expect(output).toContain('async.css')
-  })
-
-  it('creates a global layout CSS bundle', () => {
-    expect(output).toContain('page-kit-layout-styles.css')
-  })
-
-  it('creates a manifest file', () => {
-    expect(output).toContain('manifest.json')
+  it('creates the expected JS, CSS, and manifest files', () => {
+    expected.forEach((file) => {
+      expect(output).toContain(file)
+    })
```

Comment:
this still wouldn't catch unexpected files. `expect(output.sort()).toEqual(expected.sort())` would work